skeleton/nix/default.nix

```diff
@@ -0,0 +1,92 @@
+{ pkgs
```

Comment:
no preference here, but do we prefer default.nix to pkgs.nix?

Reply:
Maybe `nix/pkgs.nix` is better.